packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -175,6 +175,12 @@
 		&:active {
 			color: $white;
 		}
+
+		// Make sure the button has no hover style when it's disabled.
+		&[aria-disabled="true"]:hover {
+			color: $white;
+		}
```

Comment:
Do we also need to remove `:focus` styles?

Reply:
Could this be grouped with the `:active` rule above? E.g., 

```scss
		&:active,
		&[aria-disabled="true"]:hover {
			color: $white;
		}
```

Reply:
It looks like `:focus` doesn't have `color: initial;` that this is overriding for `:hover`.

Reply:
`:active` is also not making a difference:


https://user-images.githubusercontent.com/1157901/200921694-8ec3ff98-16b8-41b1-9e60-3f60123a0e6e.mov